.github/workflows/e2e-vitest-scenarios.yaml

```
@@ -1117,6 +1117,96 @@ jobs:
           retention-days: 14
 
+  # Focused coverage slice for the provider-routed Model Router inference
+  # contract. The retained legacy bash lane remains the source for full
+  # closeout until a later PR proves replacement and deletes it.
+  model-router-provider-routed-inference-vitest:
+    needs: [validate-jobs, generate-matrix]
+    if: ${{ (inputs.jobs == '' && inputs.scenarios == '') || contains(format(',{0},', inputs.jobs), ',model-router-provider-routed-inference-vitest,') || contains(format(',{0},', inputs.scenarios), ',model-router-provider-routed-inference,') }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 45
+    env:
+      DOCKER_CONFIG: ${{ runner.temp }}/docker-config-model-router-provider-routed-inference
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)